#!/usr/bin/env python
PACKAGE = "servo_reconfig"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Speed_Servo", double_t, 0, "Speed Servo Value", 0.0, 0, 360)
gen.add("Steer_Servo", double_t, 0, "Steering Servo Value", 0.0, 0, 360)
gen.add("Send_Message", bool_t, 0, "Check to Send Messages" , False)


exit(gen.generate(PACKAGE, "servo_reconfig", "Servo"))
